packages/components/src/CircularProgress/CircularProgress.scss

```diff
@@ -3,7 +3,9 @@ $tc-circular-progress-light-fill: #eee !default;
 $tc-circular-progress-small: 1.2rem;
 $tc-circular-progress-regular: 2rem;
 $tc-circular-progress-large: 4rem;
+$tc-circular-progress-larger: 7em;
 $tc-circular-progress-stroke-width: 5;
+$tc-circular-progress-stroke-width-lgr: 2;
```

Comment:
`lgr` should be spelled out completely as `large`

Reply:
fixed